tests/test_framework/mod.rs

```diff
@@ -533,7 +532,7 @@ impl TestFramework {
         Arc<DirectoryServer>,
         JoinHandle<()>,
     ) {
-        setup_logger(log::LevelFilter::Info);
+        setup_logger(log::LevelFilter::Info, None);
```

Comment:
This should be called after creating the temp folder in the line below. Then pass the temp folder.